human-biomolecular-data/data-sources.md

```diff
@@ -19,93 +25,134 @@ training:
+  - [Federated European Genome-phenome Archive](https://www.covid19dataportal.org/federated-ega), which provides secure controlled access sharing of sensitive patient and research subject data sets relating to COVID-19.
+  - [COVID-19 Data Portal](https://www.covid19dataportal.org/), which brings together and continuously updates relevant COVID-19 datasets and tools, will host sequence data sharing and will facilitate access to other SARS-CoV-2 resources.
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+  You can find further information about the Covid-19 Data Portal in the link [here](https://rdmkit.elixir-europe.org/covid19_data_portal).
 
-Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here. Add links to RDMkit in the metadata above, if any are needed. 
+## Data access and transfer
 
-### Existing approaches
-
-Mention existing appreaches related to the topic.
+Sharing genetic and molecular information between researchers and institutions is essential for gaining a better understanding of human biology and disease. This is especially important when it comes to infectious diseases. By studying the genetic makeup of pathogens and how they interact with human cells, researchers can identify new targets for treatments and vaccines. They can also develop strategies to prevent or contain potential outbreaks before they occur.
 
-### Licenses
+Of course, sharing this kind of sensitive information comes with challenges. Privacy, security, and ethical considerations must be taken into account. Researchers need to handle this information responsibly and with respect for individuals' rights. Legal and regulatory barriers can also impede data sharing and collaboration.
 
-<!--- Example of topic, optional --->
-
-Under what licenses do the data sources work?
+However, the benefits of sharing human biomolecular data outweigh the risks. Access to this data is crucial for scientific progress and medical advancements. It's important for us to continue finding ways to responsibly and securely share this valuable resource. Let's work together to make sure that we do so while respecting everyone's rights and ensuring that data is kept safe and secure.
 
 ### Considerations
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+When looking for solutions to human biomolecular data access, you should consider the following aspects:
+- Check with your institution that you are following the ethical and legal requirements.
+- Deposit your data in a controlled access repository
```

Comment:
I think this is not necessarily the best approach. It depends on whether the data has been pseudo-anonymised or anonymised, and primarily on the given consent and usage of the data.

Not always we should hide data behind controlled access barriers when not strictly necessary. In most cases it will be the sensible choice, but recommending it by default could be a mistake

Reply:
I would replace it by something like "...choose an adequate repository for your data..."

Reply:
This change has been addressed